cognite/extractorutils/extraction_pipelines.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from functools import wraps
+from threading import Event, Thread
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes import ExtractionPipelineRun
+
+
+def add_extraction_pipeline(
+    extraction_pipeline_ext_id: str,
+    cognite_client: CogniteClient,
+    logger: logging.Logger = None,
```

Comment:
Passing in loggers like this is not something we do in the rest of the library, I think we should create a new one in here and use that. 

So add this at the top of the module and use it instead:

``` python
_logger = logging.getLogger(__name__)
```

Reply:
Wouldn't it be convenient to pass your own logger so that it behaves similar to the rest of your code? However, I am all for conformity so I will be happy to change this, just want to check :)

Reply:
IMO it's better to let frameworks define their own loggers, and configure the logging framework instead if you want control over what is actually logged from frameworks :)

Reply:
Agree. Thanks for pointing this out.